src/index.js

```diff
@@ -0,0 +1,6 @@
+exports.say = function(name){
+    if(name==null)
+        return 'Hello world!';
+    else
+        return 'Hello '+name+'!';
```

Comment:
Add spaces between operators `Hello ' + name + '!'`
You can also rewrite this line using ES6 template strings (you have to use backticks). Something like:
`` return `Hello ${name}!` ``